01_fib/kaizhu/main.go

```diff
@@ -0,0 +1,42 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+)
+
+var out io.Writer = os.Stdout
+
+func main() {
+	fib(7)
+}
+
+func fib(n int64) {
+
+	switch {
+	// print 0 if n is 0
+	case n == 0:
+		fmt.Fprintln(out, 0)
+	// print a message if n is too big or too small
+	case n < -92 || n > 92:
+		fmt.Fprintln(out, "Please enter a number within the range [-92, 92].")
```

Comment:
Return an error in this case.

Reply:
I don't think this is an error. Instead, this should be a message remainder.